.github/workflows/build.yml

```diff
@@ -143,10 +120,71 @@ jobs:
             - name: Run Psalm
               run: ./vendor/bin/psalm --output-format=github
 
+    build-phar:
+        needs: tests
```

Comment:
```suggestion
```

Reply:
I have no issue at all with that change.

Reply:
I though made the publish-stage dependend on the tests as well as on the phar-tests so that no untested phar will be published.